starfish/test/full_pipelines/api/test_merfish.py

```diff
@@ -239,11 +235,11 @@ def test_merfish_pipeline_cropped_data():
         norm_order=2,
         crop_size=(0, 40, 40)
     )
-    spot_intensities, prop_results = psd.run(low_passed)
+    spot_intensities, prop_results = psd.run(scaled_image)
```

Comment:
Question here: what is the average intensity (float value) across the `scaled_image` before and after this change? I would scale the threshold used in the PixelSpotDetector by this value, and it might (or might not 🙃) bring the number of counts back down towards the number of spots_passing_filters.